Automattic/jetpack
# Update Pricing table design to current version

#### Changes proposed in this Pull Request:


* Changed design to match the current one here: TTzDk3dzct551hYWRXPFa3-fi-3908%3A48536
* Added option to show discount percentage
* Added option to add popover info to each item. This can be done by providing `tooltipInfo` and optoional `tooltipTitle` props to the PricingTabelItem component.
* Switched up prices, now the current one is on the left.
* Added Terms of Service note to the bottom of the table.
* Now only the main column should have a white background, this can set by adding the `primary` prop to a PricingTableColumn.

#### Other information:


#### Jetpack product discussion



1202844140176868-as-1202944779121698

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






* Fire up storybook with running `npm run storybook:dev` from the `js-packages/storybook` folder.
* Play with the added options described in the proposed changes section
* Check with Figma file if design matches.
